templates/drupal-fix-permissions.sh.erb

```diff
@@ -112,6 +112,11 @@ find . -type f -exec chmod u=rw,g=r,o= '{}' \;
 if [[ "$drupal_version" -eq 8 ]]; then
   printf "Changing permissions of "vendor/bin" directories in "${d8_vendor}" to "rwxr-----"...\n"
   chmod u+x ${d8_vendor}/bin/* || true
+
+  # Other exceptions if they exist.
+  if [ -f ${d8_vendor}/drupal/console/bin/drupal ]; then
```

Comment:
Why is this not covered by the usual vendor bin stuff?

Reply:
Theres a `bin/` directory in the webroot of D8 sites, but no `vendor/bin/` directory. Maybe we change 114 to be:

```cmd
chmod u+x ${d8_vendor}/* || true
```